resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerStateMachine.java

```diff
@@ -279,8 +279,20 @@ public EventPublisher onStateTransition(EventConsumer<CircuitBreakerOnStateTrans
         }
 
         @Override
-        public EventPublisher onStateReset(EventConsumer<CircuitBreakerOnResetEvent> onStateResetEventConsumer) {
-            registerConsumer(CircuitBreakerOnResetEvent.class, onStateResetEventConsumer);
+        public EventPublisher onReset(EventConsumer<CircuitBreakerOnResetEvent> onResetEventConsumer) {
```

Comment:
I think we don't need special event handlers like `onReset`, `onDisable` and `onForceOpen`. I would suggest to use `onStateTransition` for all state transitions.

Reply:
only for `onReset`